### PR TITLE
#9 Feat: Error Handling In C#

### PR DESCRIPTION
**Pull request by VimalkumarSelvakumar**
This assignment involves exploration on error handling and good error handling practices.
This is the task sheet for reference
👇
[Error Handling in CSharp.docx](https://github.com/VimalkumarSelvakumar/CSharpAssignments/files/12518676/Error.Handling.in.CSharp.docx)

### CODE REVIEW

File: src/Assignment9/Utility.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        public static void ChangeConsoleBackGroundColor(string color)
+        {
+            if (Enum.TryParse(color, out ConsoleColor c))
+            {
+                Console.ForegroundColor = c;
+            }
+        }
+
+        /// <summary>
+        /// Handles unhanded exception in the current application domain.
```
Comment (GowthamKarunakaran) [inline src/Assignment9/Utility.cs:58]: Correct the spelling,
`unhanded ->unhandled`
Check for any other spell (or) grammar mistakes and correct it

File: src/Assignment9/Utility.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        /// </summary>
+        /// <param name="sender">Object that raise the exception.</param>
+        /// <param name="e">Details of the exception object.</param>
+        public static void UnhandedExceptionTrapper(object sender, UnhandledExceptionEventArgs e)
```
Comment (GowthamKarunakaran) [inline src/Assignment9/Utility.cs:62]: unhanded ->unhandled

**Review by GowthamKarunakaran — APPROVED** [review]
Resolve the given comments and proceed with completing this PR.

File: src/Assignment9/Utility.cs (hunk 1 above)
Comment (VimalkumarSelvakumar) [reply src/Assignment9/Utility.cs:58]: Sorry for the confusion.
The intelliSense suggest this spelling instead of `unhandled`

**Pull request merged**